suite-native/atoms/src/Button/Button.tsx

```diff
@@ -156,6 +157,19 @@ export const Button = ({
         onPressColor,
     );
 
+    const animatedButtonStyles = [
+        animatedPressStyle,
+        applyStyle(buttonStyle, {
+            size,
+            colorScheme,
+            isDisabled,
+        }),
+        style,
+    ];
+
+    // Remove animation if disabled.
+    if (isDisabled) animatedButtonStyles.shift();
```

Comment:
These code is duplicated over multiple buttons, better just pass `isDisabled` to `useButtonPressAnimatedStyle` a there simply do:

```
if(isDisabled) return;

return animatedStyle;
```

Undefined is valid value for style too.

Reply:
fixed in e44e7fee9